style.css

```diff
@@ -103,3 +108,270 @@ footer h3 {
   width: 25%;
   margin-right: 60%;
 }
+
+#working {
+  margin: 50px 0 57px 0;
+}
+
+.project {
+  background-color: #fff;
+  border-radius: 2.5%;
+  margin-bottom: 10vh;
+}
+
+.project_image {
+  margin-top: 32px;
+  align-self: start;
+  margin-left: 24px;
+  max-width: calc(100% - 48px);
+  min-height: 110px;
+}
+
+.general {
+  width: 90%;
+  margin-left: 25px;
+}
+
+.general footer {
+  display: flex;
+  flex-direction: column;
+}
+
+.general_inf {
+  display: flex;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+
+  /* font-size: 13px; */
+  margin-top: 5px;
+  margin-left: 0;
+}
+
+.project_description {
+  font-family: poppins, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 5px;
+  text-align: justify;
+  margin-right: 30px;
+}
+
+.skills {
+  display: flex;
+  align-items: baseline;
+  justify-content: flex-start;
+  padding-left: 0;
+  overflow: auto;
+}
+
+.skills li {
+  background-color: #ebebfe;
+  border-radius: 25%;
+  margin-right: 10px;
+  font-family: poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  list-style: none;
+  flex: 1;
+  text-align: center;
+}
+
+.external_link {
+  border: 1px solid #396df2;
+  border-radius: 5px;
+  width: 45%;
+
+  /* margin-top: 16px;
+  padding: 10px 0; */
+  text-align: center;
+  margin-left: 16px;
+  margin-bottom: 32px;
+}
+
+.external_link a {
+  text-decoration: none;
+  font-family: poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #396df2;
+  text-align: center;
+}
+
+.external_link:hover {
+  background-color: #6070ff;
+  border-radius: 8px;
+  box-shadow: 0 8px 16px rgba(64, 84, 253, 0.24);
+}
+
+.external_link:visited {
+  background: #2230d2;
+  border-radius: 8px;
+}
+
+.external_link a:hover { color: #fff; }
```

Comment:
- Great job implementing button styling when the button is hovered👍. However currently there is no button styling for when the button is pressed. You can achieve this using the `:active` pseudo selector. Please make sure to follow the guidelines provided by the figma template🙏.

<img width="252" alt="image" src="https://user-images.githubusercontent.com/37116322/219719493-00a15265-602e-4b5c-92cd-c9f631e71676.png">

- Also, currently the text of your button only changes to white when the mouse is directly over the text and not the full button. Please fix this so that the text changes to white whenever the mouse hovers anywhere in the button😎